partner_auto_salesman/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = 'res.partner'
+
+    user_id = fields.Many2one('res.users',
```

Comment:
This field already exists upstream, so don't redefine everything; just the things that change (actually, the default value).